remote_engine_client/src/config.rs

```diff
@@ -46,6 +48,8 @@ impl Default for Config {
             route_cache_max_size_per_partition: 16,
             route_cache_partition_num: 16,
             compression: CompressOptions::default(),
+            max_retry: 5,
+            retry_interval: ReadableDuration::from_str("5s").unwrap(),
```

Comment:
I guess there more simple way to build such readable duration:
```suggestion
            retry_interval: ReadableDuration::seconds(5)
```